src/components/RenderRouter/HomeChurchItem.tsx

```diff
@@ -214,6 +228,17 @@ export class ContentItem extends React.Component<Props, State> {
     }
   }
 
+  private daysOfWeek = [
```

Comment:
I think it makes more sense if the day of week is not pluralized. @deeinstereo thoughts?

Reply:
Just realized you can't seen the lines below. The options for day of week are currently Sundays, Mondays, etc.